### PR TITLE
made sure data navigation only loads once

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**

### CODE REVIEW

File: husky_components/navigation/main.js [hunk 1]
```diff
@@ -982,6 +992,8 @@ define(function() {
          * Remove data navigation
          */
         removeDataNavigation: function() {
+            if (!!this.dataNavigationIsLoading) return;
```
Comment (danrot) [inline husky_components/navigation/main.js]: Why is that necessary? Shouldn't it be possible to remove the data navigation when it is currently loading?

**Comment by marcelmoosbrugger** [comment]
Closed in favor of https://github.com/massiveart/husky/pull/685
